app/services/bufferizer/listings_tweet.rb

```diff
@@ -0,0 +1,22 @@
+module Bufferizer
+  class ListingsTweet
+    def self.call(listing, tweet, _admin_id = nil)
```

Comment:
I think I'd just remove the last argument instead of ignoring it and update the call site. They are distinct services, they don't necessarily have to have the same public interface.

Reply:
Good catch! Making our linters work overtime, I didn't even do this 😅